packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.spec.js

```diff
@@ -64,27 +67,21 @@ describe('Component > InteractionLayer', function () {
         ],
         type: 'drawing'
       })
-      function InteractionLayerContainer () {
-        const [ activeMark, setActiveMark ] = useState(null)
-        return (
-          <InteractionLayer
-            activeMark={activeMark}
-            activeTool={activeTool}
-            frame={2}
-            setActiveMark={setActiveMark}
-            height={400}
-            width={600}
-          />
-        )
-      }
+      const setActiveMarkStub = sinon.stub().callsFake(() => mockMark)
       activeTool = mockDrawingTask.activeTool
       sinon.stub(activeTool, 'createMark').callsFake(() => mockMark)
-      wrapper = mount(
-        <SVGContext.Provider value={{ svg, getScreenCTM }}>
-          <svg>
-            <InteractionLayerContainer />
-          </svg>
-        </SVGContext.Provider>
+      wrapper = shallow(
+        <InteractionLayer
+          activeMark={mockMark}
+          activeTool={activeTool}
+          frame={2}
+          setActiveMark={setActiveMarkStub}
+          height={400}
+          width={600}
+        />, {
+          wrappingComponent: SVGContext.Provider,
```

Comment:
Do you need to wrap the component in a context provider if you're mocking `useContext`?

Reply:
This is cruft  I think.